odoo/addons/base/models/res_users.py

```diff
@@ -1641,7 +1641,8 @@ def _prepare_warning_for_group_inheritance(self, user):
             missing_implied_groups = missing_implied_groups.filtered(
                 lambda g:
                 g.category_id not in (group.category_id | categories_to_ignore) and
-                g not in current_groups_by_category[g.category_id]
+                g not in current_groups_by_category[g.category_id] and
```

Comment:
Hello @praw-odoo ,

Since the initial issue is already fixed, can you reword your commit message to just explain we hide the Other category if the user is not in debug mode.

Something like this:
```txt
[FIX] base: display `Other` category in warning message if debug=false

Before this commit, when the user changes the access rights of another
user, he will see a warning message the other access rights will change
with his changes. In that warning message, `Other` category could be
displayed and that category is only displayed in the user form view if
the user is in debug mode since it is a Technical category.

This commit hides the `Other` category in the warning message when the
user is not in debug mode to be consistent with the visibility of that
category in the form view.

task-3049636
```

Reply:
Done Thanks :+1: